app/javascript/driving/components/ActiveRide.js

```diff
@@ -72,6 +80,18 @@ const renderButtons = props => {
           >
             Cancel Ride
           </button>
+          <p className="text-center">
+            Cancelling a ride will make it available to other drivers
+          </p>
+          <button
+            className="btn btn-outline btn-api m-t-md"
+            onClick={() => archiveRide(ride)}
+          >
+            Archive Ride
+          </button>
+          <p className="text-center">
+            Archive a ride when a rider has already voted.
+          </p>
```

Comment:
It would be nice to semantically attach these "labels" to the buttons they label. I don't have an idea in mind. Perhaps something we can come back to later.